builtin/common/settings/components.lua

```diff
@@ -442,6 +442,51 @@ local function make_noise_params(setting)
 	}
 end
 
+function make.key(setting)
+	local btn_bind = "bind_" .. setting.name
+	local btn_clear = "unbind_" .. setting.name
+	return {
+		info_text = setting.comment,
+		setting = setting,
+
+		get_formspec = function(self, avail_w)
+			self.resettable = core.settings:has(setting.name)
+			local btn_bind_width = math.max(2.5, avail_w/2)
+			local value = core.settings:get(setting.name)
+			local fs = {
+				("label[0,0.4;%s]"):format(get_label(setting)),
```

Comment:
These setting descriptions are no longer translated since this PR, despite the use of gettext. I tried to find an explanation for this without success so far...

Reply:
The reason is simply that the description of the settings in `settingtypes.txt` are different from their `guiKeyChangeMenu` counterparts.

(That said, I might change some to verify that the `gettext` call works.)

Reply:
I changed some in https://github.com/luanti-org/luanti/pull/15791/commits/06e05e0dea3cfc8d9c7f8be314c363af9af3ca37. I decided to change some other a bit differently since I redesigned the components to make labels take up the full width of the tab (instead of just half of it) and this allows for longer descriptions.